### DISCUSSION TITLE
Which model can I use to extract woodwinds?

### DISCUSSION THREAD

**Discussion by pillpillpill** [Anjok07/ultimatevocalremovergui#540] [Q&A]
Hi!
Is there any model to extract woodwinds? I try to use htdemucs_6s but it send woodwinds in others with other instruments. Lalal AI have an option to extract winds

**Comment by ghost** [comment]
A "wind instruments" model is arealdy available : "**17_HP-Wind_Inst-UVR**" (grab it in the Download Center)
